docs/alerts/webhook-connections/aws-lambda.md

```diff
@@ -13,54 +13,51 @@ You can trigger an AWS Lambda function directly from a Monitor or Scheduled Sear
+In more demanding use cases, lambda can be accessed via **API Gateway**. To use this approach, generate an Invoke URL with a POST method for your Lambda function by creating an API in Amazon API Gateway. For information about exposing an HTTP endpoint in API Gateway, see the [Amazon API Gateway documentation](https://docs.aws.amazon.com/apigateway/latest/developerguide/getting-started.html).
 
 Secure your Lambda Function URL or API Gateway method by selecting **AWS_IAM** for the authorization type in configuration on AWS side.
 
 :::tip
-Having URL to the webhook, copy and paste it into a notepad. You will need it to configure the webhook connection in the next section.
+Have your webhook URL handy by copying and pasting it to a notepad. You'll need it to configure your webhook connection in the next section.
 :::
 
-## Create an IAM user with required privileges
-
 ### Lambda Function URL
 
-If you use the Lambda Function URL, create an IAM user with **lambda:InvokeFunctionUrl** action allowed. 
-
-:::caution
-Required action is different than the *lambda:InvokeFunction* used in AWS managed IAM Policy *AWSLambdaRole* so one needs to create *Customer managed* role
-:::
-
-<details><summary>Template for the IAM Policy</summary>
-Custom policy: 
-```json
-{
-    "Version": "2012-10-17",
-    "Statement": [
-        {
-            "Effect": "Allow",
-            "Action": [
-                "lambda:InvokeFunctionUrl"
-            ],
-            "Resource": "arn:aws:lambda:*:*:function:*"
-        }
-    ]
-}
-```
-
-</details>
-
-### API Gateway
-
-If you use API Gateway, create an IAM user who has basic API gateway invoke access. You can use the AWS managed policy **AmazonAPIGatewayInvokeFullAccess**.
-
-:::tip
-Having user account with required IAM policy assigned create AWS Access Key for this account and note **Access key** and **Secret access key** to authenticate Webhook connection in next section.
-:::
+If you use the Lambda Function URL to expose a Lambda function, you'll need to:
+
+1. Create an IAM user with the **lambda:InvokeFunctionUrl** action allowed.
+  :::note
+  This is different from the **lambda:InvokeFunction** action, which is used in the AWS-managed IAM Policy **AWSLambdaRole**.
+  :::
+2. Create and assign **Customer managed** policy.
+   ```json title="IAM Customer Managed Policy template"
+   {
+       "Version": "2012-10-17",
+       "Statement": [
+           {
+               "Effect": "Allow",
+               "Action": [
+                   "lambda:InvokeFunctionUrl"
+               ],
+               "Resource": "arn:aws:lambda:*:*:function:*"
+           }
+       ]
+   }
+   ```
+3. Create an AWS Access Key for this account. Take note of your **Access key** and **Secret access key**, which you'll need in the next section to authenticate your Webhook connection.<br/><img src={useBaseUrl('img/connection-and-integration/customer-managed-policy.png')} alt="Customer managed policy" />
```

Comment:
I've shared picture to give you some context. Not sure if we should include this in our doc.